src/proto/FunctionRpc.proto

```diff
@@ -315,9 +327,18 @@ message TypedData {
     CollectionString collection_string = 9;
     CollectionDouble collection_double = 10;
     CollectionSInt64 collection_sint64 = 11;
+    SharedMemoryData shared_memory_data = 12;
   }
 }
 
+// Used to provide metadata about shared memory region to read/write data
+message SharedMemoryData {
```

Comment:
I think the needs to be more generic:  `RpcMemoryMappedFile` ?

Reply:
This is not the file itself, just metadata about it. And since Memory Maps are not backed by files on Windows, to keep things simple, let's call it `RpcMemoryMapInfo`?